lib/components/artist/__tests__/__snapshots__/artworks-tests.js.snap

```diff
@@ -74,109 +74,16 @@ exports[`test renders properly 1`] = `
       </Text>
     </Text>
     <ArtworksGrid
-      artworks={undefined}
-      onComplete={[Function]}
-      queryArtworksKeypath="artist.artworks"
-      queryForPage={[Function]}
-      queryState={
+      artist={
         Object {
-          "availability": "IS_FOR_SALE",
-        }
-      } />
-  </View>
-  <View
-    style={
-      Array [
-        Object {
-          "backgroundColor": "#e5e5e5",
-          "height": 1,
-          "marginLeft": -20,
-          "marginRight": -20,
-        },
-        Object {
-          "marginBottom": 20,
-          "marginTop": 40,
-        },
-      ]
-    } />
-  <View>
-    <Text
-      accessible={true}
-      allowFontScaling={true}
-      ellipsizeMode="tail"
-      numberOfLines={1}
-      style={
-        Array [
-          Object {
-            "fontSize": 17,
-          },
-          Object {
-            "marginBottom": 20,
-          },
-          Object {
-            "fontFamily": "AGaramondPro-Regular",
+          "counts": Object {
+            "artworks": 5,
+            "for_sale_artworks": 2,
           },
-        ]
-      }>
-      <Text
-        accessible={true}
-        allowFontScaling={true}
-        ellipsizeMode="tail"
-        numberOfLines={1}
-        style={
-          Array [
-            Object {
-              "fontSize": 17,
-            },
-            Object {
-              "fontSize": 20,
-            },
-            Object {
-              "fontFamily": "AGaramondPro-Regular",
-            },
-          ]
-        }>
-        Other Works
-      </Text>
-       
-      <Text
-        accessible={true}
-        allowFontScaling={true}
-        ellipsizeMode="tail"
-        numberOfLines={1}
-        style={
-          Array [
-            Object {
-              "fontSize": 17,
-            },
-            Array [
-              Object {
-                "fontSize": 20,
-              },
-              Object {
-                "color": "#666666",
-              },
-            ],
-            Object {
-              "fontFamily": "AGaramondPro-Regular",
-            },
-          ]
-        }>
-        (
-        3
-        )
-      </Text>
-    </Text>
-    <ArtworksGrid
-      artworks={undefined}
-      onComplete={null}
-      queryArtworksKeypath="artist.artworks"
-      queryForPage={[Function]}
-      queryState={
-        Object {
-          "availability": "IS_NOT_FOR_SALE",
         }
-      } />
+      }
+      filter="IS_FOR_SALE"
+      onComplete={[Function]} />
```

Comment:
I guess it's the removal of showing all it's child components, this test now just lets us know the components props come in